<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputMovie</class>
 <widget class="QDialog" name="inputMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="durationLayout">
    <item>
     <widget class="QLabel" name="durationLabel">
      <property name="text">
       <string>Movie Duration (in minutes):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="duration">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="maximum">
       <number>400</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>461</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="genre">
    <item>
     <widget class="QLabel" name="genreLabel">
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="genreList">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Action</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comedy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Romance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Horror</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kids and Family</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Documentary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Indie</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>461</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="year">
    <item>
     <widget class="QLabel" name="yearLabel">
      <property name="text">
       <string>Release Year:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="yearBox">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximumDate">
       <date>
        <year>2023</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>1950</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input Movie</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tiltle">
    <item>
     <widget class="QLabel" name="movieLabel">
      <property name="text">
       <string>Movie Title:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="movieTitle">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
